ACE/ace/Log_Record.cpp

```diff
@@ -19,6 +19,14 @@
 
 #include "ace/OS_Memory.h"
 
+// The sprintf format needs to be different for Windows and POSIX
```

Comment:
Looks like this part would be useful in ace_wchar.h or OS_NS_stdio.h since other callers of ACE_OS::snprintf may need it.

Reply:
Which name is better for it?

* `ACE_TEXT_FORMAT_SPECIFIER`
* `ACE_STRING_FORMAT_SPECIFIER`
* `ACE_STR_FORMAT_SPECIFIER`
* `ACE_C_STRING_FORMAT_SPECIFIER`
* `ACE_C_STR_FORMAT_SPECIFIER`

Reply:
This was added in #1907

Reply:
Looks I remembered correctly that there was a macro already, it was just not merged yet, thanks for linking Adam